### DISCUSSION TITLE
news/release-9.0/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#595] [News]
# news/release-9.0/

Do it all, everything!

https://prismlauncher.org/news/release-9.0/

**Comment by KTrain5169** [comment]
9.0 RELEASE?! YAY!!

**Comment by A2R14N** [comment]
Is it supposed to force restart my computer? i had unsaved work...

**Comment by Agonabi** [comment]
hello. I have a problem, I can't update prism launcher to the newest update. it says that it can't be updated if it is running from the same location it was downloaded to. can someone help me ?

**Comment by hgeri06** [comment]
This is my favorite launcher and finally 9.0 released!

Keep up the good work devs!
Huge respect :)

**Comment by StrongSand94191** [comment]
holy shit automatic java management is great, ty devs
